ext/zip/tests/oo_archive_flag.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+ZipArchive::getArchiveFlag and setArchiveFlag
+--SKIPIF--
+<?php
+if (!extension_loaded('zip')) die("skip zip extension not available");
```

Comment:
```suggestion
--EXTENSIONS--
zip
--SKIPIF--
<?php
```

The EXTENSIONS section is used to check (and load) extensions are available.

Reply:
yes, but not for pecl  (which is compatible with old php versions) ;)

Reply:
Right, but you should still be able to have the extensions section, old version of run-test will ignore them AFAIK

Reply:
btw, fixed  (I will revert to old way for pecl)